### PR TITLE
refactor: Radio component

### PR DESCRIPTION
**Pull request by tinaClin**
這次refactor有幾個重要的地方：
- Color style
https://github.com/trendmicro-frontend/styled-ui/blob/master/packages/react-styled-ui/src/Radio/styles.js#L14-L34
image

改成
https://github.com/trendmicro-frontend/styled-ui/blob/refactor/radio-component/packages/react-styled-ui/src/Radio/styles.js#L13-L37
image

- Update docs after HIE reviewed (Task: https://task.jarvis.trendmicro.com/browse/HSU-51)
- 其他bug fixing (Task: https://task.jarvis.trendmicro.com/browse/HSU-51)

### CODE REVIEW

File: packages/react-styled-ui/src/Radio/index.js [hunk 1]
```diff
@@ -7,13 +7,13 @@ import VisuallyHidden from '../VisuallyHidden';
 import useRadioStyle from './styles';
 
 const sizes = {
-  lg: '20px',
+  lg: '24px',
```
Comment (cheton) [inline packages/react-styled-ui/src/Radio/index.js]: 問題同 Checkbox

File: packages/react-styled-ui/src/Radio/index.js [hunk 2]
```diff
@@ -7,13 +7,13 @@ import VisuallyHidden from '../VisuallyHidden';
...
   md: '16px',
   sm: '12px',
 };
 
 const iconSizes = {
-  lg: '10px',
+  lg: '12px',
```
Comment (cheton) [inline packages/react-styled-ui/src/Radio/index.js]: 問題同 Checkbox

Comment (tinaClin) [reply packages/react-styled-ui/src/Radio/index.js]: 4的倍數可以調整，但 iconSize = '6px' 就沒有辦法了...
```
const iconSizes = {
  lg: '3x',
  md: '2x',
  sm: '6px',
};
```
寫這樣子會不會不好看？

Comment (cheton) [reply packages/react-styled-ui/src/Radio/index.js]: 對! 不如不要這樣寫

Comment (cheton) [reply packages/react-styled-ui/src/Radio/index.js]: 在想要不要乾脆把 rem 通通換回 px ?

Comment (tinaClin) [reply packages/react-styled-ui/src/Radio/index.js]: 我想`iconSize`改用計算的方式(size/2)，不要這樣子hardcode好了~

Comment (cheton) [reply packages/react-styled-ui/src/Radio/index.js]: `calc(${size} / 2)`

File: packages/react-styled-ui/src/Radio/index.js (hunk 1 above)
Comment (tinaClin) [reply packages/react-styled-ui/src/Radio/index.js]: Fixed in [afa8f44]

File: packages/react-styled-ui/src/Radio/index.js (hunk 2 above)
Comment (tinaClin) [reply packages/react-styled-ui/src/Radio/index.js]: Fixed in [afa8f44]

**Review by cheton — APPROVED** [review]

**Pull request merged**